rholang/src/main/scala/coop/rchain/rholang/interpreter/accounting/Costs.scala

```diff
@@ -39,12 +40,12 @@ trait Costs {
       "equality check"
     )
 
-  final val BOOLEAN_AND_COST = Cost(2)
-  final val BOOLEAN_OR_COST  = Cost(2)
+  final val BOOLEAN_AND_COST = Cost(2, "boolean and")
```

Comment:
This should be part of the first commit.

Alternatively, a commit log allowing separation of those concerns would be:

```
Add `operation` field to cost (remove the ability to pass no/empty operation name, combine operation names for * - +)

Add a logging variant of the `_cost` effect (uses FunctorTell[F, Chain[Cost]]) underneath
```